lib/tictactoe/console_io.ex

```diff
@@ -17,7 +17,22 @@ defmodule TicTacToe.ConsoleIO do
     |> output()
   end
 
-  def turn_message(%Player{name: name, type: :human}), do: output("#{name}'s turn:")
+  def player_menu(options) do
+    output("Who would you like to play against?")
+
+    options_with_number = Enum.with_index(options, 1)
+
+    for {{label, _game}, number} <- options_with_number do
+      output("#{number} - #{label}")
+    end
+  end
+
+  def turn_message(%Player{name: name, type: :ai}) do
+    output("#{name} is processing its move:\n")
+    :timer.sleep(800)
```

Comment:
is this timer necessary? is there a timing issue if this doesnt exist?

Reply:
Should I take the timer out?

Reply:
Ah, cool! I didn't consider that as a possibility. Will update now!